content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -195,4 +195,27 @@ openssl command is expected.
     $ kubectl delete ns foo istio-system
     {{< /text >}}
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy for the foo namespace:
+
+    {{< text bash >}}
+    $ kubectl delete peerauthentication -n foo default
+    {{< /text >}}
+
+*   Remove the sample and httpbin application:
+
+    {{< text bash >}}
+    $ kubectl delete -f samples/sleep/sleep.yaml -n foo
+    $ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+    {{< /text >}}
+
+*   Remove the Istio Components from the cluster:
```

Comment:
```suggestion
*   Remove Istio from the cluster:
```

I'm inclined to suggest that verbs in each of the bullets be updated to follow the verb in the command, example for this change Remove to Uninstall.

Reply:
So the recommendation is to say:
- Uninstall sample applications `httpbin and `sleep`
- Uninstall Istio from the cluster